### PR TITLE
Create models

### PR DESCRIPTION
**Pull request by anneli-pixel**
Created models with validations and associations as well as a basic seed file.

### CODE REVIEW

File: app/models/song.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Song < ApplicationRecord
+  belongs_to :teacher
+  has_many :videos, dependent: :destroy
+  has_many :lyrics, dependent: :destroy
+  # has_many :lyric_chords, through: :lyrics
```
Comment (sofialovin) [inline app/models/song.rb:5]: maybe these two could be uncommented out? I feel like we will probably need has many chords at some point

Comment (anneli-pixel) [reply app/models/song.rb:5]: Sure! How do I do this? I change it in my branch, open another pull request and close this one 🤔 ?

Comment (anneli-pixel) [reply app/models/song.rb:5]: Ah, I'm just realising I can do it in here!

Comment (sofialovin) [reply app/models/song.rb]: Ah you can? I don't know exactly how but great. I am done. You can merge when you're ready

**Pull request merged**